samples/web/content/apprtc/apprtc_test.py

```diff
@@ -96,6 +103,246 @@ def testJoinAndLeave(self):
     response = self.makePostRequest('/join/' + room_id)
     caller_id = self.verifyJoinSuccessResponse(response, True, room_id)
     self.makePostRequest('/leave/' + room_id + '/' + caller_id)
+    
+  def addTestData(self):
```

Comment:
The test file is really long now. Probably move the new tests to a separate file.